ui/app/helpers/format-ttl.js

```diff
@@ -0,0 +1,22 @@
+import { helper } from '@ember/component/helper';
+
+export default helper(function formatTtl([timestring], { removeZero = false }) {
+  // Expects a number followed by one of s, m, or h
+  // eg. 40m or 1h20m0s
+  let matches = timestring?.match(/([0-9]+[h|m|s])/g);
```

Comment:
nice regex work

Reply:
Thank you! Took me way longer than I'd like to admit 🙈 😂